listenbrainz/db/user.py

```diff
@@ -213,3 +213,18 @@ def update_latest_import(musicbrainz_id, ts):
             except sqlalchemy.exc.ProgrammingError as e:
                 logger.error(e)
                 raise DatabaseException
+
+
+def get_recently_logged_in_users():
+    """Returns a list of users who have logged-in in the last X days, X is
```

Comment:
you can say 
Returns a list of users who have logged-in in the last config.STATS_CALCULATION_LOGIN_TIME days